integrations/access/accessrequest/app.go

```diff
@@ -431,6 +431,10 @@ func (a *App) updateMessages(ctx context.Context, reqID string, tag pd.Resolutio
 		return trace.Wrap(err)
 	}
 
+	if err := a.bot.NotifyUser(ctx, reqID, reqData); err != nil {
+		return trace.Wrap(err)
```

Comment:
All the NotImplemented errors from the non-Slack plugins will bubble up here.

We might want to catch them to avoid polluting logs as they are expected.